### ISSUE TITLE
Alternative text is not provided for Graphs images.

### ISSUE THREAD

**Issue by msftedad** [FormidableLabs/victory#2635]
**Describe the bug**
Alternative text is not provided for Graphs images.
The Edge stats graphs should be having accessible name assigned.
Screen reader Users fails to get exact information while navigating through Graphics Stats charts.

**Victory version**
Which npm version are you using?
0.26.1

**Code Sandbox link**
Please include a code sandbox link or a similar reproduction if possible.
[Victory | VictoryLegend (formidable.com)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2f%2fformidable.com%2fopen-source%2fvictory%2fdocs%2fvictory-legend%2f&data=05%7c01%7cv-srgoll%40microsoft.com%7ca98a21c4bd20408fb0f008db15ede6c6%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638127883766699290%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=trncDRVhZVXjjDtKG6EkuBKJYlYfDz%2fy6GwE4VycLOI%3D&reserved=0)
[Victory | VictoryChart (formidable.com)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2f%2fformidable.com%2fopen-source%2fvictory%2fdocs%2fvictory-chart&data=05%7c01%7cv-srgoll%40microsoft.com%7ca98a21c4bd20408fb0f008db15ede6c6%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638127883766699290%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=TY2kqEFeqiam8TJlPlO%2BdSSXmLVsVmQ15lONyifG%2foo%3D&reserved=0)
**To Reproduce**
1. Open the Above URL.
2. Verify whether the Alternative text is provided for the Graphs images or not.

**Expected behavior**
The Victory graph images should have alternative text information to meet Accessibility compliant standards.

**Screenshots**
If applicable, add screenshots to help explain your problem. (You can exclude this if you included a reproduction)

**Desktop (please complete the following information):**
 - OS: Windows11 Enterprise version: 22621.1992
 - Browser: Edge
 - Version: 115.0.1901.188 (Official build) (64-bit)
 - Application: [Issues | Perf Lab | Edge Team](https://edgeteam.ms/perf-lab/issues/61a683c61174e211002246ab/bisections?bisectionId=%22623a51a1dee8c46fa4c04b92%22)
![Alternative text](https://github.com/FormidableLabs/victory/assets/93735775/6d388775-b07f-4f1f-bd72-d77fccbb5aa5)
![alt](https://github.com/FormidableLabs/victory/assets/93735775/71cd4af2-34ee-455b-9c4e-78c7d8a72575)

**Comment by msftedad** [comment]
Hi @msabree, Any update on this issue?

**Comment by carbonrobot** [comment]
@msftedad Are you looking for the aria-label property? https://formidable.com/open-source/victory/docs/new#accessibility-features

**Comment by msftedad** [comment]
Hi @carbonrobot, Issue is that graph and images do not have alt-text. Check the guideline to understand the requirement [Understanding Success Criterion 1.1.1: Non-text Content | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Comment by carbonrobot** [comment]
Thanks @msftedad, I believe we already support that by the following techniques, suggested later in the doc.

[ARIA6: Using aria-label to provide labels for objects](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6) and
[ARIA10: Using aria-labelledby to provide a text alternative for non-text content](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA10) as well as [ARIA15: Using aria-describedby to provide descriptions of images](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA15) for long form content.

Example:

```
<VictoryBar
    horizontal
    style={{ data: { fill: "#c43a31", opacity: 0.9 } }}
    data={accessibilityGroupData.b}
    groupComponent={
      <VictoryAccessibleGroup
        aria-label="victory bar group 2"
        desc="accessible bar chart group 2"
        aria-describedby="accessible bar chart group 2 aria description"
        tabIndex={67}
      />
    }
  />
```

This applies a `<g>` tag to the rendered chart and assigns an `aria-label` and `aria-describedby` field for accessibility.
